tools/mydumper.md

```diff
@@ -0,0 +1,47 @@
+---
+title: mydumper 使用文档 
+summary: 使用 mydumper 从 TiDB 导出数据。
+category: tools
+---
+
+# mydumper 使用文档
+
+## mydumper 简介
+
+`mydumper` 是 [mydumper](https://github.com/maxbube/mydumper) 的 fork 项目，并添加了一些针对 TiDB 的功能。推荐使用此工具对 TiDB 进行逻辑备份。
+
+[下载 Binary](http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz)。
+
+## 相比于普通的 mydumper，此工具有哪些改进之处?
+
++ 使用 `tidb_snapshot` 而非 `FLUSH TABLES WITH READ LOCK` 提供备份一致性 
+
```

Comment:
For such a list without sublist or detailed explanation, the blank line between each item is unnecessary. You can just remove it.